crates/macros/src/lib.rs

```diff
@@ -60,6 +60,8 @@ pub fn build(stream: proc_macro::TokenStream) -> proc_macro::TokenStream {
             );
 
             path.push("windows.rs");
+            println!("cargo:rerun-if-changed={}", path.to_str().expect("`OUT_DIR` not a UTF-8 string"));
```

Comment:
I'm not clear why we need this. If the .windows folder changes the build script reruns and causes the OUT_DIR to be updated. That's the output. We watch the input, not the output.

Reply:
It makes it easier to regenerate `windows.rs` without a full `cargo clean` (loosing potentially large intermediates); just modifying or deleting `windows.rs` (`rm target/**/windows.rs`) is enough to trigger a regen. However, this doesn't seem to debounce properly with the build script writing to the file at the same time, and always reruns it instead (also when printing after writing). Removed it, should be fine to modify the file containing `build!()` too.